lametro/views.py

```diff
@@ -739,6 +739,8 @@ def search(self):
             # Don't auto-escape my query! https://django-haystack.readthedocs.io/en/v2.4.1/searchqueryset_api.html#SearchQuerySet.filter
             sqs = sqs.filter_or(attachment_text=Raw(self.cleaned_data['q']))
 
+        sqs = sqs.filter(viewable=True)
```

Comment:
if we always hide bills, should we index them at all?

Reply:
(i.e., do we have all the info we need to omit them when we build the index?)

Reply:
(i forget, does our current pipeline update the index with every scrape, or with enough frequency for this suggestion to make sense?)

Reply:
I asked myself this very question! 

I decided to index "hidden" bills for one reason – [the integrity script](https://github.com/datamade/django-councilmatic/blob/master/councilmatic_core/management/commands/data_integrity.py), which checks that the Councilmatic database and the Solr index have the same number of bills. If we do not index a "hidden" bill, then this script will complain about a missing bill in the Solr index. I think it's easier to modify the SearchQuerySet, rather than finding a work around with how the integrity script or `import_data` works. 

(N.B. `data_integrity` runs after every execution of `update_index`, which runs after `import_data` which runs [four times per hour.](https://github.com/datamade/la-metro-councilmatic/blob/master/scripts/lametro-crontasks))

Reply:
ha, great minds. :-) that makes sense to me. maybe it would be good a drop a comment or docstring in the `search_indexes.py` file?